sources/Component/Controllable/ControllableComponent.hpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by Tom Augier on 2021-05-20.
+// Edited by Benjamin Henry on 2021-05-20.
+//
+
+#pragma once
+
+#include "lib/wal/sources/Component/Component.hpp"
+#include "lib/wal/sources/Entity/Entity.hpp"
+
+namespace BBM 
+{
+	class ControllableComponent : public WAL::Component
+	{
+	private:
+		float _moveX = 0;
+		float _moveZ = 0;
+		bool _jump = false;
+		bool _bomb = false;
+		bool _pause = false;
```

Comment:
The pause is not something that can be controlled. A controllable component allows an entity to move, not handle every input. The pause should be handled by another component/system (I believe we will have a "GameManager" entity containing components that manage the whole game state like the pause)